switch crashdb to debian
Index: apport/etc/apport/crashdb.conf
===================================================================
--- apport.orig/etc/apport/crashdb.conf	2014-11-23 14:47:59.751629696 +0530
+++ apport/etc/apport/crashdb.conf	2014-11-23 14:47:59.751629696 +0530
@@ -1,6 +1,6 @@
 # map crash database names to CrashDatabase implementations and URLs
 
-default = 'debug'
+default = 'debian'
 
 databases = {
     'ubuntu': {
