Index: apport/etc/default/apport
===================================================================
--- apport.orig/etc/default/apport	2012-07-17 16:12:27.689194779 +0530
+++ apport/etc/default/apport	2012-07-17 16:22:29.912181042 +0530
@@ -2,3 +2,12 @@
 # you can temporarily override this with
 # sudo force_start=1 /etc/init.d/apport start
 enabled=1
+
+# These settings are mandatory for apport to work with
+# the Debian BTS
+#
+# Sender email address
+#sender=user@domain.com
+#
+# SMTP Host
+smtphost=reportbug.debian.org
