display debug symbol package name
Index: apport/apport/crashdb_impl/debian.py
===================================================================
--- apport.orig/apport/crashdb_impl/debian.py	2015-01-14 15:07:37.505821689 +0530
+++ apport/apport/crashdb_impl/debian.py	2015-01-14 15:09:05.549821040 +0530
@@ -46,8 +46,14 @@
             report['UnreportableReason'] = 'Please configure sender settings in /etc/apport/crashdb.conf'
 
         # At this time, we are not ready to take CrashDumps
+        try:
+            buggyPackage = report['Package'].split(' ')[0]
+            buggyVersion = report['Package'].split(' ')[1]
+        except (KeyError, ValueError):
+            buggyPackage = "UNKNOWN"
+
         if 'Stacktrace' in report and not report.has_useful_stacktrace():
-            report['UnreportableReason'] = 'Incomplete backtrace. Please install the debug symbol packages'
+            report['UnreportableReason'] = 'Incomplete backtrace. Please install the debug symbols for package %s' % (buggyPackage)
 
         return apport.crashdb.CrashDatabase.accepts(self, report)
 
@@ -62,7 +68,8 @@
 
         # Frame the report in the format the BTS understands
         try:
-            (buggyPackage, buggyVersion) = report['Package'].split(' ')
+            buggyPackage = report['Package'].split(' ')[0]
+            buggyVersion = report['Package'].split(' ')[1]
         except (KeyError, ValueError):
             return False
 
