display debug symbol package name
Index: apport/apport/crashdb_impl/debian.py
===================================================================
--- apport.orig/apport/crashdb_impl/debian.py	2015-01-14 13:35:23.993862513 +0530
+++ apport/apport/crashdb_impl/debian.py	2015-01-14 13:36:52.685861859 +0530
@@ -46,8 +46,14 @@
             report['UnreportableReason'] = 'Please configure sender settings in /etc/apport/crashdb.conf'
 
         # At this time, we are not ready to take CrashDumps
+        try:
+            buggyPackage = report['Package'].split(' ')[0]
+            buggyVersion = report['Package'].split(' ')[1]
+        except (KeyError, ValueError):
+            buggyPackage = "UNKNOWN"
+
         if 'Stacktrace' in report and not report.has_useful_stacktrace():
-            report['UnreportableReason'] = 'Incomplete backtrace. Please install the debug symbol packages'
+            report['UnreportableReason'] = 'Incomplete backtrace. Please install the debug symbols for package %s' % (buggyPackage)
 
         return apport.crashdb.CrashDatabase.accepts(self, report)
 
