Fix my stupid assumption on apport's 'Package' field
Index: apport/apport/crashdb_impl/debian.py
===================================================================
--- apport.orig/apport/crashdb_impl/debian.py	2012-11-18 13:57:15.192796001 +0530
+++ apport/apport/crashdb_impl/debian.py	2012-11-20 12:29:01.881650814 +0530
@@ -63,6 +63,8 @@
         # Frame the report in the format the BTS understands
         try:
             (buggyPackage, buggyVersion) = report['Package'].split(' ')
+	except ValueError:
+            report['UnreportableReason'] = 'Invalid Apport Report. The report has been wrongly modified by an external apport hook'
         except KeyError:
             return False
 
