Index: apport/apport/crashdb_impl/debian.py
===================================================================
--- apport.orig/apport/crashdb_impl/debian.py	2012-11-18 13:43:27.640654734 +0530
+++ apport/apport/crashdb_impl/debian.py	2012-11-18 13:47:08.884687185 +0530
@@ -62,7 +62,7 @@
 
         # Frame the report in the format the BTS understands
         try:
-            (buggyPackage, buggyVersion) = report['Package'].split(' ')
+            (buggyPackage, buggyVersion) = report['Package'], report['PackageVersion']
         except KeyError:
             return False
 
