don't hardcode dpkg-divert path

Debian Bug #742106
Index: apport/backends/packaging-apt-dpkg.py
===================================================================
--- apport.orig/backends/packaging-apt-dpkg.py	2014-03-22 12:38:36.588254838 +0530
+++ apport/backends/packaging-apt-dpkg.py	2014-03-22 12:45:33.884246596 +0530
@@ -337,7 +337,7 @@
         instead of the one from the current system.
         '''
         # check if the file is a diversion
-        dpkg = subprocess.Popen(['/usr/sbin/dpkg-divert', '--list', file],
+        dpkg = subprocess.Popen(['dpkg-divert', '--list', file],
                                 stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         out = dpkg.communicate()[0].decode('UTF-8')
         if dpkg.returncode == 0 and out:
