Index: apport/apport/crashdb_impl/debian.py
===================================================================
--- apport.orig/apport/crashdb_impl/debian.py	2012-12-21 19:12:35.557986548 +0530
+++ apport/apport/crashdb_impl/debian.py	2012-12-31 18:08:36.187907752 +0530
@@ -98,7 +98,7 @@
         msg.add_header('Usertag', 'apport-%s' % report['ProblemType'].lower())
 
         s = smtplib.SMTP(self.options['smtphost'])
-        s.sendmail(self.options['sender'], self.options['recipient'], msg.as_string())
+        s.sendmail(self.options['sender'], self.options['recipient'], msg.as_string().encode('UTF-8') )
         s.quit()
 
     def get_comment_url(self, report, handle):
